relayer/crates/starknet-integration-tests/src/tests/ics20.rs

```diff
@@ -517,14 +522,6 @@ fn test_starknet_ics20_contract() -> Result<(), Error> {
         let starknet_ics20_send_message = {
             let current_starknet_time = starknet_chain.query_chain_status().await?.time;
 
-            let denom = PrefixedDenom {
-                trace_path: vec![TracePrefix {
-                    port_id: ics20_port.to_string(),
-                    channel_id: starknet_channel_id.to_string(),
-                }],
-                base: Denom::Hosted(denom_cosmos.to_string()),
-            };
-
```

Comment:
btw, can we keep the call to `ibc_token_address` query endpoint to make sure we are storing the correct address ? it will be an assert right now -- as we are pre-creating the erc20 contract.

Reply:
Brought it back [d3dde23](https://github.com/informalsystems/ibc-starknet/pull/326/commits/d3dde2357d8edbff1dfe94dd2ca48326691616c2)